pypika/enums.py

```diff
@@ -46,6 +46,8 @@ class Boolean(Comparator):
 class Order(Enum):
     asc = "ASC"
     desc = "DESC"
+    asc_nulls_last = "ASC NULLS LAST"
```

Comment:
This isn't supported in all DB types, it works in Postgres though, not sure if this should even be here